specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/examples/CreateADiskEncryptionSet.json

```diff
@@ -0,0 +1,63 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2020-12-01",
+    "diskEncryptionSetName": "myDiskEncryptionSet",
+    "diskEncryptionSet": {
+      "location": "West US",
+      "identity": {
+        "type": "SystemAssigned"
+      },
+      "properties": {
+        "activeKey": {
+          "sourceVault": {
+            "id": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.KeyVault/vaults/myVMVault"
+          },
+          "keyUrl": "https://myvmvault.vault-int.azure-int.net/keys/{key}"
+        },
+        "encryptionType": "EncryptionAtRestWithCustomerKey"
+      }
+    }
+  },
+  "responses": {
+    "202": {
+      "body": {
```

Comment:
>"body": { [](start = 6, length = 9)

RP should always return top level properties like Type and Id.

Reply:
It won't require an api-version change since the schema for this version already defines Type and Id.

Reply:
the diskrp server side code doesn't return it currently.. so this would need a code change and deployment